module/batch-action/src/Infrastructure/Handler/CreateBatchActionCommandHandler.php

```diff
@@ -47,5 +48,7 @@ public function __invoke(CreateBatchActionCommand $command): void
             $entryCommand = new ProcessBatchActionEntryCommand($id, $type, $resourceId);
             $this->commandBus->dispatch($entryCommand, true);
         }
+        $endCommand = new EndBatchActionProcessCommand($id, $type);
```

Comment:
Why You send `command` and next handle this command and send `event` ? 
I thing in this place You may send event  `BatchActionEndedEvent`